models/DataObject/Fieldcollection/Dao.php

```diff
@@ -143,9 +143,6 @@ public function load(DataObject\Concrete $object): array
      */
     public function delete(DataObject\Concrete $object, bool $saveMode = false): array
     {
-        // empty or create all relevant tables
-
-        /** @var DataObject\ClassDefinition\Data\Fieldcollections $fieldDef */
         $fieldDef = $object->getClass()->getFieldDefinition($this->model->getFieldname(), ['suppressEnrichment' => true]);
```

Comment:
to fix phpstan error, we might need to restore this line

```suggestion
        /** @var DataObject\ClassDefinition\Data\Fieldcollections $fieldDef */
        $fieldDef = $object->getClass()->getFieldDefinition($this->model->getFieldname(), ['suppressEnrichment' => true]);
```

Reply:
thanks @kingjia90. I was about to commit that later today ;)